bootstrap.sh

```diff
@@ -1 +1,5 @@
 #!/bin/bash
+kubectl apply -f configMap.yml
```

Comment:
There is a typo in the filename `configMap.yml`. Ensure that the file name matches the actual file name, which should be `confgiMap.yml` according to the previous review. Correct this to avoid errors when applying the configuration.